execution/gethexec/executionengine.go

```diff
@@ -386,9 +386,9 @@ func (s *ExecutionEngine) NextDelayedMessageNumber() (uint64, error) {
 	return currentHeader.Nonce.Uint64(), nil
 }
 
-func MessageFromTxes(header *arbostypes.L1IncomingMessageHeader, hooks *arbos.SequencingHooks) (*arbostypes.L1IncomingMessage, error) {
+func MessageFromTxes(header *arbostypes.L1IncomingMessageHeader, hooks arbos.SequencingHooks) (*arbostypes.L1IncomingMessage, error) {
```

Comment:
This function can be a member function of fullSequencinghooks, so it does not need to use the interface but the full struct and no need for GetTxErrors in the interface